.cirrus.yml

```diff
@@ -74,4 +74,4 @@ fedora_selftests_task:
        - PATH=$HOME/.local/bin:$PATH LANG=en_US.UTF-8 AVOCADO_CHECK_LEVEL=0 python3 selftests/check.py  --skip static-checks
     container:
         matrix:
-          - image: quay.io/avocado-framework/avocado-ci-fedora-33
+          - image: quay.io/avocado-framework/avocado-ci-fedora-35
```

Comment:
Are we still in need of `./contrib/containers/ci/selftests/fedora-33.docker` file? If not, perhaps we could drop those files together with this commit.

Also I wouldn't mind if you squash this commit and the previous one, since they are related.